src/resultssummarypage.cpp

```diff
@@ -78,6 +96,14 @@ ResultsSummaryPage::ResultsSummaryPage(FilterAndZoomStack* filterStack, PerfPars
                                                      ki18n("Show top hotspots for %1 events."));
             });
 
+    connect(parser, &PerfParser::perLibraryDataAvailable, this,
+            [this, perLibraryModel](const Data::PerLibraryResults& data) {
+                perLibraryModel->setData(data);
+                ResultsUtil::hideEmptyColumns(data.costs, ui->topLibraryTreeView, PerLibraryModel::NUM_BASE_COLUMNS);
+                ResultsUtil::fillEventSourceComboBox(ui->eventSourceComboBox_2, data.costs,
+                                                     ki18n("Show top hotspots for %1 events."));
```

Comment:
we use `tr` in hotspot, not `ki18n`

Reply:
This functions still uses KLocalizedString so `ki18n` is needed here. I will create a pr that fixes that

Reply:
now that the `tr` change got merged, this should be `tr` again

Reply:
done